src/sdk/namespace_fs.js

```diff
@@ -2385,6 +2395,20 @@ class NamespaceFS {
         return (file_path && file_path.endsWith(config.NSFS_FOLDER_OBJECT_NAME)) && (key && key.endsWith('/'));
     }
 
+    /**
+     *
+     * @param {*} fs_context
+     * @param {string} file_path
+     * @returns {Promise<boolean>}
+     */
```

Comment:
```suggestion
    /**
     *  _is_disabled_content_dir returns true if ..... false otherwise
     * @param {nb.NativeFSContext} fs_context
     * @param {string} file_path
     * @returns {Promise<boolean>}
     */
```